src/kubernetes/server/server-config-kafka.yaml

```diff
@@ -20,6 +20,24 @@ data:
                 users:
                   admin: "{noop}admin, ROLE_MANAGE, ROLE_VIEW"
```

Comment:
I know this is not related to the PR, but I'd want to suggest bumping the permissions of the `admin` user to include all the roles. Roles listed here: http://docs.spring.io/spring-cloud-dataflow/docs/2.0.0.BUILD-SNAPSHOT/reference/htmlsingle/#configuration-security-customizing-authorization

Folks often assume this user would have all the rights, but just with ROLE_MANAGE and ROLE_VIEW, they cannot even register apps.

Reply:
added

Reply:
reverted per discussion